src/main/java/edu/greenblitz/robotName/commands/swerve/Battery/BatteryLimiter.java

```diff
@@ -0,0 +1,48 @@
+package edu.greenblitz.robotName.commands.swerve.Battery;
+
+import edu.greenblitz.robotName.subsystems.Battery;
+import edu.greenblitz.robotName.utils.GBCommand;
+import edu.wpi.first.math.filter.LinearFilter;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+import static com.sun.org.apache.bcel.internal.util.Args.require;
+
+public class BatteryLimiter extends GBCommand {
+
+    private static final int LEN_OF_AVG = 50;
+    private static final double USAGE_MIN_LEVEL = 2;
+    private Battery battery;
+    private LinearFilter voltageFilter;
+
+
+    public BatteryLimiter() {
+        voltageFilter = LinearFilter.movingAverage(LEN_OF_AVG);
+        battery = Battery.getInstance();
+        require(battery);
+    }
+
+    @Override
+    public void initialize() {
+        for (int i = 0; i < LEN_OF_AVG; i++) {
+            voltageFilter.calculate(battery.getCurrentVoltage());
+        }
+    }
+
+    @Override
+    public void execute() {
+        double currentAverageVoltage;
```

Comment:
Why is this needed if the value is only initialized AND used inside the `if` block?